src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -265,9 +265,11 @@
                                                                  (i18n/label :t/slide-to-send))
                                           :container-style     {:z-index 2}
                                           :customization-color account-color
-                                          :on-auth-success     #(rf/dispatch
-                                                                 [:wallet/send-transaction
-                                                                  (security/safe-unmask-data %)])
+                                          :on-auth-success     #(do
+                                                                  (rf/dispatch
+                                                                   [:wallet/send-transaction
+                                                                    (security/safe-unmask-data %)])
+                                                                  (rf/dispatch [:hide-bottom-sheet]))
```

Comment:
Regarding to the solution, I think we should solve the problem in a different way, if the bottom sheet is not closing it's due to an error or a failure somewhere, maybe we could dispatch a toast for the user explaining the error. 

wdyt?